elfi/examples/lotka_volterra.py

```diff
@@ -160,7 +162,12 @@ def get_model(n_obs=50, true_params=None, seed_obs=None, **kwargs):
     """
     logger = logging.getLogger()
     if true_params is None:
-        true_params = [1.0, 0.005, 0.6, 50, 100, 10.]
+        if observation_noise:
+            true_params = [1.0, 0.005, 0.6, 50, 100, 10.]
+        else:
+            true_params = [1.0, 0.005, 0.6, 50, 100, 0.]
+    elif not observation_noise and len(true_params) > 5 and true_params[5] != 0:
```

Comment:
i read between the lines here that`true_params` could also have less than six values, but then the logger message on L194-195 assumes at least six values and prints an error in case fewer are provided. do we want users to provide six values also when `observation_noise=False`?

Reply:
Good point. It's simpler to require 5 inputs when `observation_noise=False`